.github/workflows/openms_ci_matrix_full.yml

```diff
@@ -488,10 +600,16 @@ jobs:
 
           # Upload documentation FIXME
           rsync --progress -avz -e "ssh -i ~/.ssh/private.key -o StrictHostKeyChecking=no" $GITHUB_WORKSPACE/docs/ "$USER@$HOST:/Documentation/${folder}"
+
+          if [[ 'true' == ${{ inputs.mark_as_latest }} ]]; then
```

Comment:
Can we do an initial check if a release is really justified:
do_release && branch=~ release(0-9) && BASH_REMATCH matches OpenMS version in git

Then fail early? Otherwise this will all execute and you will link anything to latest

Reply:
MAybe you can add it to the changelog extraction and move it up a little. Something without too much effort.

Reply:
https://github.com/OpenMS/OpenMS/pull/7210/commits/04ce8680bbdbdbe48740509a1fa75f0ba89aa064 does what I understand you to be asking, can you double check that this is what you meant?

Reply:
yep, very cool!